lib/nightmare.js

```diff
@@ -91,6 +91,22 @@ function Nightmare(options) {
     debug('electron child process exited with code ' + code + ': ' + help[code]);
   });
 
+  Object.defineProperty(this, 'engineVersions', {
+    get: function() {
+      delete this.engineVersions; 
+      return this.engineVersions = JSON.parse(
```

Comment:
> also defaults to false

Well.  Today I learned `defineProperty` does not work like it does in my mind.  Filing that away.

> The = operator still returns the value you tried to set even if it didn’t work, so the failure isn’t always obvious (one of those not-so-great design decisions in JS).

But the assignment ought to work with `configurable: true`, yes?